wrappers/react/test/componentInternals.spec.tsx

```diff
@@ -1,146 +1,16 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { act } from '@testing-library/react';
 import { HotTable } from '../src/hotTable';
 import { HotColumn } from '../src/hotColumn';
 import {
   createSpreadsheetData,
   mockElementDimensions,
   mountComponentWithRef,
+  renderComponentWithProps,
   sleep,
 } from './_helpers';
 import { HOT_DESTROYED_WARNING } from "../src/helpers";
-import { BaseEditorComponent } from '../src/baseEditorComponent';
-import { HotRendererProps } from '../src'
-
-describe('Subcomponent state', () => {
```

Comment:
In function-based components manipulating state from the outside isn't possible, so these tests are no longer relevant